### PR TITLE
Note `from()` can query remote sources

### LINKED ISSUE #3073: Reading Enterprise 1.9 from Cloud with flux `from` is supported
The cloud docs say:

```
from() cannot retrieve data from remote InfluxDB OSS 1.x or InfluxDB Enterprise 1.x data sources (including hosted, single-tenant InfluxDB Enterprise clusters).
```

But in fact we do support giving `host` as a 1.9 Enterprise Cluster

Also username and password authentication can be done via the 'token' argument, which is hinted at here: https://docs.influxdata.com/enterprise_influxdb/v1.9/guides/query_data/#query-data-with-flux but also works with `from`, by setting `token: "username:password"`

##### Relevant URLs
https://docs.influxdata.com/influxdb/cloud/reference/flux/stdlib/built-in/inputs/from/

TODO:

- [x] @mdevy-influxdata is going to confirm the `username:password` auth working from cloud into Enterprise 1.9 .

### PR DESCRIPTION
**Pull request by pierwill**
Closes #3073.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Review by sanderson — CHANGES_REQUESTED** [review]
The change looks good, but I think we should add some details to the existing params in the doc. I'd add suggestions, but GitHub won't let me since the params weren't modified in the PR. Here are my recommendations:

```md
### bucket
Name of the bucket to query.

**InfluxDB 1.x or Enterprise**: provide an empty string (`""`)

_**Data type:** String_

### bucketID
String-encoded bucket ID to query.

**InfluxDB 1.x or Enterprise**: provide an empty string (`""`)

_**Data type:** String_

### host
URL of the InfluxDB instance to query.
_See [InfluxDB URLs](/influxdb/v2.0/reference/urls/)._

_**Data type:** String_

### org
Organization name.

**InfluxDB 1.x or Enterprise**: provide an empty string (`""`)

_**Data type:** String_

### orgID
String-encoded [organization ID](/influxdb/v2.0/organizations/view-orgs/#view-your-organization-id) to query.

**InfluxDB 1.x or Enterprise**: provide an empty string (`""`)

_**Data type:** String_

### token
InfluxDB [API token](/influxdb/v2.0/security/tokens/).

**InfluxDB 1.x or Enterprise**: If authentication is _disabled_, provide an empty string (`""`). If authentication is _enabled_, provide your InfluxDB username and password using the `<username>:<password>` syntax.

_**Data type:** String_
```

File: content/influxdb/v2.0/reference/flux/stdlib/built-in/inputs/from.md [hunk 1]
```diff
@@ -43,9 +43,6 @@ from(
 Use `from()` to retrieve data from remote InfluxDB 2.0 OSS and InfluxDB Cloud instances.
 To query remote InfluxDB sources, include the [host](#host), [token](#token), and
 [org](#org) (or [orgID](#orgid)) parameters.
-
-`from()` **cannot retrieve data** from **remote** InfluxDB OSS 1.x or InfluxDB Enterprise 1.x data sources
```
Comment (lesam) [inline content/influxdb/v2.0/reference/flux/stdlib/built-in/inputs/from.md:47]: I think we should specifically call out 1.9+ here.

File: content/influxdb/v2.0/reference/flux/stdlib/built-in/inputs/from.md [hunk 2]
```diff
@@ -41,23 +41,25 @@ from(
 {{% note %}}
 #### Query remote InfluxDB data sources
 Use `from()` to retrieve data from remote InfluxDB 2.0 OSS and InfluxDB Cloud instances.
+(InfluxDB 1.x OSS and Enterprise 1.9+ are also supported.)
```
Comment (sanderson) [inline content/influxdb/v2.0/reference/flux/stdlib/built-in/inputs/from.md:44]:
```suggestion
Use `from()` to retrieve data from remote InfluxDB OSS 1.7+, InfluxDB Enterprise 1.9+, and InfluxDB Cloud.
```

**Review by sanderson — APPROVED** [review]
One minor change and I think it's gtg 👍🏻

**Pull request merged**